tasmota/xdrv_23_zigbee_A_impl.ino

```diff
@@ -1392,8 +1422,10 @@ void ZigbeeShow(bool json)
       bool temperature_ok = device.validTemperature();
       bool humidity_ok    = device.validHumidity();
       bool pressure_ok    = device.validPressure();
+      bool time_ok        = device.validLastSeen();
+      uint32_t now        = Rtc.utc_time;
 
-      if (temperature_ok || humidity_ok || pressure_ok) {
+      if (temperature_ok || humidity_ok || pressure_ok || time_ok) {
```

Comment:
Do you have a screen shot? Time last seen should not be limited to Temp readings.

Reply:
I included the time in the second line, as between the battery status, LQI and last seen time the width of the table was growing. I mentioned moving this to the LQI line if you'd prefer.